lib/ripple/core_ext/casting.rb

```diff
@@ -5,9 +5,9 @@
 require 'active_support/core_ext/date/conversions'
 require 'active_support/core_ext/date/zones'
 require 'active_support/core_ext/date_time/conversions'
-require 'active_support/core_ext/date_time/zones'
 require 'active_support/core_ext/time/conversions'
 require 'active_support/core_ext/time/zones'
+require 'active_support/core_ext/date_and_time/zones'
```

Comment:
Was this moved due to override patterns, or by accident? Alphabetically, it'd still go where it was before, unless I'm missing something.

Reply:
This is mentioned in the commit message: "Remove active_support/core_ext/date_time/zones and replace with active_support/core_ext/date_and_time/zones. The date_time/zones
core extension previously delegated to the date_and_time/zones extension." See https://github.com/rails/rails/commit/e0deec2e68c7db74a9c8ed25d8e7eb178d30e6ca

Reply:
No, like, I got that; it was about ordering, not about the inclusion. Previously, the `require`s were alphabetical, but with this change, they aren't, and I wasn't clear why that changed.